course/auth.py

```diff
@@ -680,9 +774,35 @@ def user_profile(request):
 
     user_form = None
 
+    def has_verified_inst_id(user):
+        if not EDITABLE_INST_ID_PRE_VRF:
+            return True and user.institutional_id or False
+        else:
+            if user.institutional_id_verified:
+                return True
+            else:
+                return False
+
+    def enable_inst_input(request):
```

Comment:
Why allow editing institutional IDs just based on whether a GET parameter is present? If you want to be able to trust those inst IDs, then this makes no sense.